src/source_sampling.h

```diff
@@ -88,24 +92,53 @@ namespace pyne {
     std::vector<double> prob; /// Probabilities.
     std::vector<int> alias; /// Alias probabilities.
   };
-  
+
+  // class Source particle
+  class SourceParticle {
```

Comment:
Maybe this should be a struct??  We should check the style guide in case it cares?

Reply:
Maybe not - instead you need a method `std::vector<double> pack()` that replaces `Sampler::get_src_xyzew()`